### ISSUE TITLE
When VL compiler code changes, vega example remains the same after refreshing

### ISSUE THREAD

**Issue by kanitw** [vega/vega-editor#136]
Suppose we have a VL spec and we modify the code in VL compiler.
After the editor automatically refresh due to changes in vl.js, the vega output should update accordingly.  However, it seems like the vega output is loaded from the cache and thus is inconsistent with the updated VL compiler code.

**Comment by domoritz** [comment]
Yeah, the problem is that the editor does not rerender. @mathisonian and I discussed that this should change.

**Comment by kanitw** [comment]
- Say you have a VL spec and it does not behave correctly. 
- You edit Vega-Lite and run `npm run tsc` in Vega-Lite
- Vega Editor got automatically refreshed but still render the old Vega spec.

**Comment by domoritz** [comment]
@kanitw Yes, this should change. The editor should just rerender.
